core/GlobalState.cc

```diff
@@ -28,6 +28,32 @@ using namespace std;
 
 namespace sorbet::core {
 
+namespace {
+// Hash functions used to determine position in namesByHash.
+
+inline unsigned int _hash_mix_unique(UniqueNameKind unk, unsigned int num, unsigned int rawId) {
+    return mix(mix(num, static_cast<u4>(unk)), rawId) * HASH_MULT2 + static_cast<u4>(NameKind::UNIQUE);
+}
+
+inline unsigned int _hash_mix_constant(unsigned int id) {
+    return id * HASH_MULT2 + static_cast<u4>(NameKind::CONSTANT);
+}
+
+inline unsigned int _hash_nameref(const GlobalState &gs, NameRef nref) {
```

Comment:
What do you think about renaming these to not be prefixed with an underscore, now that they don't live in a header?

Reply:
works for me!